### PR TITLE
chatrooms: on mobile have a submit button below message when message is being edited

### PR DESCRIPTION
**Pull request by timothyclemansinsea**
I tried to edit a message on mobile only to discover I couldn't do so because mobile doesn't have shift-enter.

### CODE REVIEW

File: src/smc-webapp/editor_chat.cjsx [hunk 1]
```diff
@@ -431,6 +443,17 @@ Message = rclass
             </div>
         </Col>
 
+    is_user_editing: ->
+        other_editors = @props.message.get('editing').remove(@props.account_id).keySeq()
+        current_user = @props.user_map.get(@props.account_id).get('first_name') + ' ' + @props.user_map.get(@props.account_id).get('last_name')
```
Comment (williamstein) [inline src/smc-webapp/editor_chat.cjsx]: This is a potential weird definition of current_user... which seems to not even be used at all in this function?

File: src/smc-webapp/editor_chat.cjsx [hunk 2]
```diff
@@ -431,6 +443,17 @@ Message = rclass
             </div>
         </Col>
 
+    is_user_editing: ->
```
Comment (williamstein) [inline src/smc-webapp/editor_chat.cjsx]: In fact, this is_user_editing function isn't called anywhere.  Just delete it.

**Comment by williamstein** [comment]
I made a few line comments.

**Comment by williamstein** [comment]
Please resolve the merge conflict.

**Comment by williamstein** [comment]
This is impossible to merge.  I would recommend just trying to redo what you did in the new version of the code.

**Comment by williamstein** [comment]
Or maybe @simonluu could move this code into the new chat rewrite.

**Comment by simonluu** [comment]
see #1018

**Pull request merged**